scripts/percona-release.sh

```diff
@@ -318,11 +321,22 @@ function disable_repository {
   if [[ ${1} = all ]]; then
     _repos=${REPOSITORIES}
     for _repository in ${_repos}; do
-      disable_component ${_repository} ${2}
+      if [[ ${_repository} != "prel" ]]; then
+        disable_component ${_repository} ${2}
+      else
+          disable_component ${_repository} experimental
+          disable_component ${_repository} testing
+      fi
     done
   else
     check_specified_repo ${1}
-    disable_component ${1} ${2}
+    if [[ ${1} != "prel" ]]; then
+      disable_component ${1} ${2}
+    else
```

Comment:
It is still possible to disable prel release in case you run the following command:
`sudo percona-release disable prel`

```
[22:57:46 vagrant@localhost:~] $ ls -la /etc/yum.repos.d/
total 44
drwxr-xr-x.  2 root root 4096 Jul 28 22:57 .
drwxr-xr-x. 62 root root 4096 Jul 27 05:31 ..
-rw-r--r--.  1 root root 1991 Oct 23  2014 CentOS-Base.repo
-rw-r--r--.  1 root root  647 Oct 23  2014 CentOS-Debuginfo.repo
-rw-r--r--.  1 root root  289 Oct 23  2014 CentOS-fasttrack.repo
-rw-r--r--.  1 root root  630 Oct 23  2014 CentOS-Media.repo
-rw-r--r--.  1 root root 5394 Oct 23  2014 CentOS-Vault.repo
-rw-r--r--   1 root root  804 Jul 28 22:57 percona-prel-experimental.repo
-rw-r--r--   1 root root  759 Jul 28 22:57 percona-prel-release.repo
-rw-r--r--   1 root root  759 Jul 28 22:57 percona-prel-testing.repo

[22:57:52 vagrant@localhost:~] $ sudo percona-release disable prel
<*> All done!

[22:58:10 vagrant@localhost:~] $ ls -la /etc/yum.repos.d/
total 44
drwxr-xr-x.  2 root root 4096 Jul 28 22:58 .
drwxr-xr-x. 62 root root 4096 Jul 27 05:31 ..
-rw-r--r--.  1 root root 1991 Oct 23  2014 CentOS-Base.repo
-rw-r--r--.  1 root root  647 Oct 23  2014 CentOS-Debuginfo.repo
-rw-r--r--.  1 root root  289 Oct 23  2014 CentOS-fasttrack.repo
-rw-r--r--.  1 root root  630 Oct 23  2014 CentOS-Media.repo
-rw-r--r--.  1 root root 5394 Oct 23  2014 CentOS-Vault.repo
-rw-r--r--   1 root root  804 Jul 28 22:57 percona-prel-experimental.repo.bak
-rw-r--r--   1 root root  759 Jul 28 22:57 percona-prel-release.repo.bak
-rw-r--r--   1 root root  759 Jul 28 22:57 percona-prel-testing.repo.bak

```

This is because of the "if [[ ${2} = all ]] || [[ -z ${2} ]]; then" check in "disable_component" sub.

The same is observed if you run:
```
[23:04:20 vagrant@localhost:~] $ ls -la /etc/yum.repos.d/
total 44
drwxr-xr-x.  2 root root 4096 Jul 28 23:04 .
drwxr-xr-x. 62 root root 4096 Jul 27 05:31 ..
-rw-r--r--.  1 root root 1991 Oct 23  2014 CentOS-Base.repo
-rw-r--r--.  1 root root  647 Oct 23  2014 CentOS-Debuginfo.repo
-rw-r--r--.  1 root root  289 Oct 23  2014 CentOS-fasttrack.repo
-rw-r--r--.  1 root root  630 Oct 23  2014 CentOS-Media.repo
-rw-r--r--.  1 root root 5394 Oct 23  2014 CentOS-Vault.repo
-rw-r--r--   1 root root  804 Jul 28 23:04 percona-prel-experimental.repo
-rw-r--r--   1 root root  759 Jul 28 23:04 percona-prel-release.repo
-rw-r--r--   1 root root  759 Jul 28 23:04 percona-prel-testing.repo

[23:04:23 vagrant@localhost:~] $ sudo percona-release disable prel all
<*> All done!

[23:04:28 vagrant@localhost:~] $ ls -la /etc/yum.repos.d/
total 44
drwxr-xr-x.  2 root root 4096 Jul 28 23:04 .
drwxr-xr-x. 62 root root 4096 Jul 27 05:31 ..
-rw-r--r--.  1 root root 1991 Oct 23  2014 CentOS-Base.repo
-rw-r--r--.  1 root root  647 Oct 23  2014 CentOS-Debuginfo.repo
-rw-r--r--.  1 root root  289 Oct 23  2014 CentOS-fasttrack.repo
-rw-r--r--.  1 root root  630 Oct 23  2014 CentOS-Media.repo
-rw-r--r--.  1 root root 5394 Oct 23  2014 CentOS-Vault.repo
-rw-r--r--   1 root root  804 Jul 28 23:04 percona-prel-experimental.repo.bak
-rw-r--r--   1 root root  759 Jul 28 23:04 percona-prel-release.repo.bak
-rw-r--r--   1 root root  759 Jul 28 23:04 percona-prel-testing.repo.bak
```

Reply:
fixed